src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37429/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37429/project/issues?id=sample&pullRequest=7004&issues=AWlr9SPSM1H-wc5koADE&open=AWlr9SPSM1H-wc5koADE)

[ISSUE_KEY]: AWlr9SPSM1H-wc5koADE